frontend/src/ui/Shortcuts.tsx

```diff
@@ -0,0 +1,259 @@
+import React from "react";
+import { useTranslation } from "react-i18next";
+import { Options, useHotkeys, HotkeyCallback } from "react-hotkeys-hook";
+import {
+    LuArrowBigUp,
+    LuArrowLeft,
+    LuArrowRight,
+    LuArrowRightToLine,
+    LuArrowDown,
+    LuArrowUp,
+} from "react-icons/lu";
+import { match, screenWidthAtMost, useColorScheme } from "@opencast/appkit";
+
+import { Modal, ModalHandle } from "./Modal";
+import { COLORS } from "../color";
+import { SKIP_INTERVAL } from "./player/consts";
+import { TranslationKey } from "../i18n";
+
+
+export type ShortcutProps = {
+    keys: string,
+    translation: TranslationKey | { key: TranslationKey; options?: Record<string, unknown> };
+    options?: Options,
+}
+
+export const SHORTCUTS = {
+    general: {
+        showOverview: {
+            keys: "?",
+            translation: "shortcuts.general.show-overview",
+        },
+        closeModal: {
+            keys: "escape",
+            translation: "shortcuts.general.close-modal",
+        },
+        tab: {
+            keys: "tab",
+            translation: "shortcuts.general.tab",
+        },
+        search: {
+            keys: "s; shift+s",
+            translation: "shortcuts.general.search",
+        },
+        // Todo: moar shortcuts?
+    },
+    player: {
+        play: {
+            keys: "k; space",
+            translation: "shortcuts.player.play",
+            options: {
+                scopes: ["player"],
+                // Don't trigger when a button is focused. This way, users can still
+                // use the space bar to control other elements by default.
+                ignoreEventWhen: e => (e.key !== "k" && (
+                    e.target instanceof HTMLButtonElement
+                        || e.target instanceof HTMLInputElement
+                )),
+                // But still disable scrolling with space.
+                preventDefault: e => !(e.key !== "k" && (
+                    e.target instanceof HTMLButtonElement
+                        || e.target instanceof HTMLInputElement
+                )),
```

Comment:
This can just be `preventDefault: true`. The `ignoreEventWhen` is checked first and if that's `true`, nothing happens.